src/piped/sqs.clj

```diff
@@ -25,14 +25,14 @@
                            :VisibilityTimeout visibility-timeout}}]
     (api.async/invoke client request)))
 
-(defn change-visibility-batch [client messages visibility-timeout]
+(defn change-visibility-batch [client messages backoff-fn]
```

Comment:
Breaks backwards compatibility. Again, you could go multi arity and supply an `opts` map:
```clojure
{:nack-delay-fn (constantly 5)}
```